### PR TITLE
Added a new package idds-workflow

### PR DESCRIPTION
**Pull request by yesw2000**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by yesw2000** [comment]
@beckermr 
I added another package idds-workflow. Please help merge it. Thanks!

File: recipes/idds-workflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  host:
+    - idds-common ==0.5.2
+    - pip
+    - python
+  run:
+    - idds-common ==0.5.2
+    - pip
+    - python
```
Comment (beckermr) [inline recipes/idds-workflow/meta.yaml:24]: A few comments here:

1. The host section should only contain packages needed at install-time. My guess is that you only need python and pip there. 

2. The run section should contain only stuff needed at run-time. In this case, you should remove pip like we did last time.

3. If the version of idds-workflow is supposed to match idds-common always, you should use jinja2 like this

```
- idds-common =={{ version }}
```

File: recipes/idds-workflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: cfb19c04c716beabb9c330edf976eb5a8e23c4365094c46f21f10b2875f31512
+
+build:
+  number: 0
```
Comment (beckermr) [inline recipes/idds-workflow/meta.yaml:13]: This recipe should likely be noarch: python

File: recipes/idds-workflow/meta.yaml [hunk 3]
```diff
@@ -9,18 +9,17 @@ source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
   sha256: cfb19c04c716beabb9c330edf976eb5a8e23c4365094c46f21f10b2875f31512
 
-build:
+build: noarch: python
```
Comment (beckermr) [inline recipes/idds-workflow/meta.yaml]:
```suggestion
build:
  noarch: python
```

File: recipes/idds-workflow/meta.yaml [hunk 4]
```diff
@@ -9,18 +9,17 @@ source:
...
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - idds-common ==0.5.2
+    - idds-common =={{ version }}
```
Comment (beckermr) [inline recipes/idds-workflow/meta.yaml:19]: You likely don't need this in host per my previous comment. Only things imported by the setup.py are needed in host.

**Comment by yesw2000** [comment]
@beckermr ,

I guess that I have applied all the changes you suggested, right? And it is ready to go.

Shuwei

File: recipes/idds-workflow/meta.yaml (hunk 4 above)
Comment (beckermr) [reply recipes/idds-workflow/meta.yaml:19]: Please see this comment

File: recipes/idds-workflow/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: cfb19c04c716beabb9c330edf976eb5a8e23c4365094c46f21f10b2875f31512
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - idds-common =={{ version }}
```
Comment (beckermr) [inline recipes/idds-workflow/meta.yaml:19]:
```suggestion
```

**Comment by beckermr** [comment]
Nope sorry. Pleasew remove `idds-common =={{ version }}` from the `host` section.

**Comment by yesw2000** [comment]
@beckermr ,

Thanks for your suggestion.
Following your suggestion, I just removed the requirement of **idds-common** under the **host section**.

Best regards,
Shuwei

**Pull request merged**